perspective_reddit_bot/get_top_subreddits.py

```diff
@@ -0,0 +1,48 @@
+"""Small tool for scraping http://redditlist.com/"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import sys
+
+from  bs4 import BeautifulSoup
+import requests
+
+
+# Only getting SFW subreddits.
+BASE = 'http://redditlist.com/sfw?page='
+
+
+def fetch_page(page_num):
+    print('fetching page', page_num)
+    url = BASE + str(page_num)
+    return BeautifulSoup(requests.get(url).text, 'html.parser')
+
+
+def get_hot_list(soup):
+    # First ".span4.listing" element gets the "Recent Activity" list.
+    # The 2nd list is by subscribers, and the 3rd is by 24h growth.
+    items = soup.select('#listing-parent .span4.listing')[0].select('.listing-item')
+    return [i.get('data-target-subreddit') for i in items]
+
+
+def main():
+    outfile = sys.argv[1]
```

Comment:
Maybe use argparse for consistency and readability?

Reply:
done